packages/corejs/src/monitoring/index.tsx

```diff
@@ -24,7 +24,7 @@ export function MonitoringProvider({ children, config }: PropsWithChildren<Monit
       dsn: config.dsn,
       environment: config.environment,
       debug: config.debug,
-      tracesSampleRate: config.tracesSampleRate,
+      tracesSampleRate: config.tracesSampleRate || 0.1,
```

Comment:
use `??` operator instead of `||`

Reply:
Done.

The null coalescing operator works similarly to the || operator for fallback values. The only difference is that it makes sure that if you enter an empty string or a 0, it will use that value and not the fallback.

Reply:
Done